web_server/posts/models.py

```diff
@@ -36,19 +45,18 @@ class Post(models.Model):
     author = models.ForeignKey(Author, on_delete=models.PROTECT)
 
     # Storing a list of comma delineated categories
-    # @todo Should this be replaced with a many-to-many combo table and a separate 'categories' model?
-    # categories = models.CharField(max_length=256)
+    categories = models.ManyToManyField(Category, blank=True)
 
     # Count of comments
+    # @todo is this field strictly necessary? It should be computed not stored
     count = models.PositiveIntegerField()
 
     # Page Size
     # @todo What is a 'page'? A page of comments?
     size = models.PositiveIntegerField()
```

Comment:
This may also be computed at get time. We should pick a max number of posts per page, then compute size as count/max.

Reply:
Field removed and left a note.